Godeps/Godeps.json

```diff
@@ -1,7 +1,9 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.5",
+	"GoVersion": "go1.6",
```

Comment:
busted on what version of go I'm using. It's pointless/harmless/not used by anything at all.